### PR TITLE
[6.x] Return player spirit to correct graveyard & zone

### LINKED ISSUE #14089: [6.x.x] Incorrect Location Of Graveyards
Rev: https://github.com/TrinityCore/TrinityCore/commit/36963f244279b958d9808aa4e3a420328a53e9ef

Issue: For example you create a level 1 orc and you die, you go to barrens as a spirit, and you shouldnt go so far

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/8438498-6-x-x-incorrect-location-of-graveyards?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1310&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by walter-exit**
**Changes proposed:**

Loading of subzones from graveyard_zone table by ObjMgr::LoadGraveyardZones() so for instance dying in Northshire returns you to proper graveyard (in Northshire) instead of Westfall.

**Target branch(es):** 6.x

**Issues addressed:** Closes  #14089

**Tests performed:** Built & tested ingame

**Known issues and TODO list:**
- none

### CODE REVIEW

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -5997,11 +5997,18 @@ void ObjectMgr::LoadGraveyardZones()
             continue;
         }
 
+        
+        /*
+        JaredNinja: commented code, because we want to be able to load subzones too. So we can appoint graveyards to subzones.
```
Comment (Goatform) [inline src/server/game/Globals/ObjectMgr.cpp]: Comment not really needed or at least edit it.

File: sql/updates/world/6.x/2016_09_26_00_world.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+-- Changing subzone 6170 (Northshire, Elwynn). Was zone 12 before, not right.
+-- Part of my total fix for graveyard problems (https://github.com/TrinityCore/TrinityCore/issues/14089)
+UPDATE graveyard_zone SET GhostZone=6170 WHERE ID=105 AND Faction=469
```
Comment (Aokromes) [inline sql/updates/world/6.x/2016_09_26_00_world.sql]: newline

Comment (JaredNinja) [inline sql/updates/world/6.x/2016_09_26_00_world.sql]: can you elaborate?

**Comment by JaredNinja** [comment]
Update 1: better?

File: sql/updates/world/6.x/2016_09_26_00_world.sql (hunk 2 above)
Comment (tkrokli) [inline sql/updates/world/6.x/2016_09_26_00_world.sql]: your SQL:
```sql
UPDATE graveyard_zone SET GhostZone=6170 WHERE ID=105 AND Faction=469
```
Recommended SQL:
```sql
UPDATE `graveyard_zone` SET `GhostZone`= 6170 WHERE `ID`= 105 AND `Faction`= 469;
```
Also please add a CarriageReturn/LineFeed at the end of the last text line.
The writing cursor should always rest on a new blank line below all text files.

Comment (JaredNinja) [inline sql/updates/world/6.x/2016_09_26_00_world.sql]: Will do

**Comment by Shauren** [comment]
Merged in 9c220e8f55eaa8cfc93ed26ea7318430e5c6749b and d205a96250ac9a67900fe83e35495ac4b7754c93

**Pull request closed without merging**